pmd-visualforce/src/test/java/net/sourceforge/pmd/lang/vf/VFTestContstants.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.vf;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+public abstract class VFTestContstants {
```

Comment:
The usual pattern is constant holding classes is to create a final class with a private constructor instead of an abstract class...